builtin/providers/aws/cloudfront_distribution_configuration_structure.go

```diff
@@ -719,12 +719,15 @@ func expandCustomErrorResponse(m map[string]interface{}) *cloudfront.CustomError
 	if v, ok := m["error_caching_min_ttl"]; ok {
 		er.ErrorCachingMinTTL = aws.Int64(int64(v.(int)))
 	}
-	if v, ok := m["response_code"]; ok {
+	if v, ok := m["response_code"]; ok && v.(int) != 0 {
 		er.ResponseCode = aws.String(strconv.Itoa(v.(int)))
+	} else {
+		er.ResponseCode = aws.String("")
```

Comment:
@stack72, as @jrnt30 mentioned, this is actually necessary, because the attribute is `TypeInt`, but the API and SDK expect `string`. There's a breakdown in the conversion between @jrnt30 in the referenced issue.

The reasoning for keeping `response_code` as a `TypeInt` is three-fold: 
 * It's the HTTP response code that needs to be sent back (and hence can only be int)
 * It's that way in CloudFormation (which is what I was using as my reference implementation as much as possible)
 * `cloudfront.CustomErrorResponse.ErrorCode` (the incoming error code) is `*int64`, so it's kind of odd that this would be a `*string`.

As such, we have to set this to `""`, as the zero value of `int`, `0`, converts to `"0"` with `strconv.Itoa()`, versus an empty string.